lib/tasks/wishlist.rake

```diff
@@ -0,0 +1,21 @@
+namespace :wishlist do
+  desc 'Populate settings with wishlist domain extension whitelist'
+  task 'create_extension_whitelist' => :environment do
+    # Wishlist supported domain extensions
+    domain_extensions_description = <<~TEXT.squish
+      Supported domain extensions for wishlist domain monitoring.
+    TEXT
+
+    extensions = ['ee', 'pri.ee', 'com.ee', 'med.ee', 'fie.ee']
+    domain_extensions = Setting.new(
+      code: :wishlist_supported_domain_extensions,
+      value: extensions,
+      description: domain_extensions_description
+    )
+    if domain_extensions.save
```

Comment:
Fail fast (`save!`) approach might be better here. This way you get meaningful description right away instead of just "Failed", if something goes wrong. `puts` might be eliminated entirely in this case.

Also I think it's important to denote that is is a temporary data migration, by putting it under "data_migrations" namespace (and folder), for example.